apps/web/test/lib/getSchedule.test.ts

```diff
@@ -281,7 +302,7 @@ afterEach(async () => {
 
 describe("getSchedule", () => {
   describe("User Event", () => {
-    test("correctly identifies unavailable slots from Cal Bookings", async () => {
+    test.skip("correctly identifies unavailable slots from Cal Bookings", async () => {
```

Comment:
@hariombalhara I'm disabling these tests for now. We need to mock deeper and don't rely on a DB for unit testing. That's what E2E is for.